docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for ModelMesh Serving using the KServe Python SDK\n",
+    "\n",
+    "## This notebook shows how to use the KServe SDK to create, update, and delete an InferenceService\n",
+    "\n",
+    "- This sample assumes ModelMesh Serving was deployed using the [quickstart guide](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md)."
```

Comment:
no need for a single bullet point. we are not listing something here

Reply:
@lgdeloss noticed you used a header style for the change here, I think you can just use regular text for the quickstart guide line (i.e. no `#`s)